### PR TITLE
HM_2 Sergey_Kravchuk

### PR DESCRIPTION
**Pull request by SergeyKravchuk**

### CODE REVIEW

File: Sergey_Kravchuk [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (pluhin) [inline Sergey_Kravchuk:1]: please remove this file, all files should be inside your folder

File: Sergey_kravchuk/02.Git.Local/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# git clone https://github.com/SergeyKravchuk/sa.it-academy.by
```
Comment (pluhin) [inline Sergey_kravchuk/02.Git.Local/README.md:1]: task is not finished

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Sergey_Kravchuk (hunk 1 above)
Comment (SergeyKravchuk) [reply Sergey_Kravchuk:1]: не вижу этот файл через Bash
image

File: Sergey_kravchuk/02.Git.Local/README.md (hunk 2 above)
Comment (SergeyKravchuk) [reply Sergey_kravchuk/02.Git.Local/README.md:1]: write please what else I have to do

File: Sergey_kravchuk/02.Git.Local/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# 265  cd c/Users/s.kravchuk/
```
Comment (pluhin) [inline Sergey_kravchuk/02.Git.Local/README.md:1]: This history is history of your commands how prepare home work, but not home work itself

I need list commands, how you did this: https://github.com/pluhin/sa.it-academy.by/wiki/02.-GIT.-Local

**Review by pluhin — CHANGES_REQUESTED** [review]
to many commits in one PR, please squash them to one
https://www.git-tower.com/learn/git/faq/git-squash#:~:text=In%20case%20you%20are%20using,option%20from%20the%20contextual%20menu.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+#  #############DESCRIPTION OF MY PROJECT######################
```
Comment (pluhin) [inline README.md:1]: To many commits, and changes not in your files

**Review by pluhin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**